.dependabot/config.yml

```diff
@@ -0,0 +1,127 @@
+version: 1
+
```

Comment:
Can you add a comment here with the link to the config file docs? https://dependabot.com/docs/config-file/

Reply:
Yes, that I can do easily :stuck_out_tongue: 
Good idea! :blush: